lib/pf/Authentication/Source/LDAPSource.pm

```diff
@@ -61,6 +61,7 @@ has 'stripped_user_name' => (isa => 'Str', is => 'rw', default => 'yes');
 has '_cached_connection' => (is => 'rw');
 has 'cache_match' => ( isa => 'Bool', is => 'rw', default => 0 );
 has 'email_attribute' => (isa => 'Maybe[Str]', is => 'rw', default => 'mail');
+has 'realm' => (isa => 'ArrayRef[Str]', is => 'rw');
```

Comment:
Since that field seems common to all sources, it should be moved in Authentication/Source.pm